api/src/jvmMain/kotlin/Routes.kt

```diff
@@ -122,6 +126,10 @@ class Routes {
             class TestsToRun(val parent: Summary)
         }
 
+        @Location("/tests")
+        class Tests(val build: Build)
+
+        //TODO remove after changes on the frontend
```

Comment:
should we add Jira task because we wait for a few tasks? For example, "remove after changes on the frontend EPMDJ-<Front-end task>"